packages/tomls/src/share_token/share_tokens/rselini.toml

```diff
@@ -0,0 +1,105 @@
+[var.rselini_vars]
+rseliniTokenName = "Reya Lm Selini"
+rseliniTokenSymbol = "rSelini"
+rseliniTokenDecimals = "18"
+
+[clone.rselini]
+source = "<%= settings.shareTokensProxyPackage %>"
+target = "reya-share-tokens:1.0.0@rselini"
+options.owner = "<%= settings.owner %>"
+options.salt = "<%= settings.rseliniSalt %>"
+depends = ["var.initial_proxy_vars", "var.cannonClonePackages"]
+
+# Upgrades implementation of InitialProxy to point to the new
+# router implementation.
+[invoke.upgrade_rselini_proxy]
+target = ["rselini.InitialProxy"]
+fromCall.func = "owner"
+func = "upgradeTo"
+args = ["<%= imports.reyaShareTokenRouter.contracts.ShareTokenRouter.address %>"]
+factory.RSELINIProxy.abiOf = ["reyaShareTokenRouter.ShareTokenRouter"]
+factory.RSELINIProxy.event = "Upgraded"
+factory.RSELINIProxy.arg = 0
+depends = ["clone.rselini"]
+
+[invoke.rselini_initialize]
+target = ["RSELINIProxy"]
+fromCall.func = "owner"
+func = "initialize"
+args = [
+    "<%= settings.rseliniTokenName %>",
+    "<%= settings.rseliniTokenSymbol %>",
+    "<%= settings.rseliniTokenDecimals %>",
+]
+depends = ["invoke.upgrade_rselini_proxy"]
+
+[invoke.rselini_unpause]
+target = ["RSELINIProxy"]
+fromCall.func = "owner"
+func = "setFeatureFlagAllowAll"
+args = ["<%= keccak256(hexlify('global')) %>", true]
+depends = ["invoke.upgrade_rselini_proxy"]
+
+[invoke.rselini_set_pausers]
+target = ["RSELINIProxy"]
+fromCall.func = "owner"
+func = "setDeniers"
+args = [
+    "<%= keccak256(hexlify('global')) %>",
+    [
+        "<%= settings.pauser1 %>",
+        "<%= settings.pauser2 %>",
+        "<%= settings.pauser3 %>",
+        "<%= settings.pauser4 %>",
+    ],
+]
+depends = ["invoke.upgrade_rselini_proxy"]
+
+[invoke.rselini_global_config]
+target = ["RSELINIProxy"]
+fromCall.func = "owner"
+func = "setGlobalConfiguration"
+args = [
+    { coreProxy = "<%= contracts.CoreProxy.address %>" },
+]
+depends = ["invoke.upgrade_rselini_proxy"]
+
+[invoke.rselini_enable_subscriber]
+target = ["RSELINIProxy"]
+fromCall.func = "owner"
+func = "addToFeatureFlagAllowlist"
+args = [
+    "<%= keccak256(hexlify('subscription')) %>",
+    "<%= settings.rseliniSubscriber1 %>",
+]
+depends = ["invoke.upgrade_rselini_proxy"]
+
+[invoke.rselini_enable_redeemer]
+target = ["RSELINIProxy"]
+fromCall.func = "owner"
+func = "addToFeatureFlagAllowlist"
+args = [
+    "<%= keccak256(hexlify('redemption')) %>",
+    "<%= settings.rseliniRedeemer1 %>",
+]
+depends = ["invoke.upgrade_rselini_proxy"]
+
+[invoke.rselini_enable_custodian]
+target = ["RSELINIProxy"]
+fromCall.func = "owner"
+func = "addToFeatureFlagAllowlist"
+args = [
+    "<%= keccak256(hexlify('whitelistedCustodians')) %>",
```

Comment:
small one, maybe should rename this in the code to authorizedCustodians (given that's the language we use in the code) for consistency, don't think need to do that now but up to you ser @0xZenus

Reply:
same goes for whitelistedTokens

Reply:
I see that these are the names used for feature flags in reya-network. https://github.com/Reya-Labs/reya-network/blob/bb47f26f9f3bc165247bc9708fbc4a67ccc2d0ea/share-token/src/libraries/FeatureFlagSupport.sol#L17

Reply:
i know, i was suggesting we just chane it there as well, but not an issue really